src/index.ts

```diff
@@ -130,7 +130,7 @@ class Monika extends Command {
 
     'create-config': Flags.boolean({
       description:
-        'Create config from HAR (-H), postman (-p), insomnia (-I) export file, or open Monika Configuration Generator using default browser',
+        'Create config from HAR (-H), postman (-p), insomnia (-I), sitemap (-s) export file, or open Monika Configuration Generator using default browser',
```

Comment:
-s is stun instead? is sitemape --sitemap?

Reply:
updated